packages/core/src/routes/connector.ts

```diff
@@ -193,10 +193,9 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
         validateConfig(config);
       }
 
-      // FIXME @Darcy [LOG-4696]: revisit databaseMetadata check when implementing AC
-
-      const connector = await updateConnector({ set: body, where: { id }, jsonbMode: 'replace' });
-      ctx.body = { ...connector, metadata, type };
+      await updateConnector({ set: body, where: { id }, jsonbMode: 'replace' });
+      const connector = await getLogtoConnectorById(id);
+      ctx.body = transpileLogtoConnector(connector);
```

Comment:
Nit(Not realted to this PR):
I found the `POST /connectors` handler does not return the correct connector data type. Maybe add an issue to track this or update it later.

Reply:
<img width="443" alt="image" src="https://user-images.githubusercontent.com/10806653/205236874-0f13a0b2-a241-4f52-b3d1-31ce25c05909.png">

Reply:
LOG-4857